### PR TITLE
completed three column challenge

### PR DESCRIPTION
**Pull request by NolanReda**
three column

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Nolan! A couple notes:
1. There is too much vertical space between the images and the headline beneath it
2. The text in each column is too close to the text in the adjacent columns

Please revise and submit a new screenshot with your revision

**Comment by NolanReda** [comment]
three column

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.column-third {
+  margin: auto;
+  flex-basis: calc(100% / 3);
+}
+
+.border {
+  justify-content: center;
+  align-items: center;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:51]: This isn't a flexbox, so these aren't doing anything. If you add `display: flex` to this rule, they will have an effect, and you will be able to remove the margin from your `img` elements.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by NolanReda** [comment]
three column

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**